corehq/apps/app_manager/static/app_manager/js/releases.js

```diff
@@ -398,7 +398,7 @@ hqDefine('app_manager/js/releases.js', function () {
                     } else if (self.lastAppVersion() !== self.currentAppVersion()) {
                         self.actuallyMakeBuild();
                     } else {
-                        window.alert(gettext("No new changes to deploy!"));
+                        window.alert(gettext("No new changes!"));
```

Comment:
gahh, this alert is so annoying. Not saying you should change it here, just venting 😝